src/app/pages/account/users/user-form/user-form.component.ts

```diff
@@ -388,7 +388,7 @@ export class UserFormComponent implements FormConfiguration {
 
     /* list groups */
     this.ws.call('group.query').pipe(untilDestroyed(this)).subscribe((groups) => {
-      this.loader.callDone.emit(status);
+      this.loader.callDone.emit(window.status);
```

Comment:
> The status property of the Window interface was originally intended to set the text in the status bar at the bottom of the browser window.
I don't see how this would be relevant here. Looks like a mistake and needs deeper investigation.